sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/FileSystemApiTests.java

```diff
@@ -2215,6 +2216,40 @@ public void perCallPolicy() {
             .getHeaders().getValue(X_MS_VERSION));
     }
 
+    @Test
+    public void defaultAudience() {
+        DataLakeFileSystemClient aadFsClient =
+            getFileSystemClientBuilderWithTokenCredential(ENVIRONMENT.getPrimaryAccount().getDataLakeEndpoint())
+                .fileSystemName(dataLakeFileSystemClient.getFileSystemName())
+                .audience(null) // should default to "https://storage.azure.com/"
+                .buildClient();
+
+        assertTrue(aadFsClient.exists());
+    }
+
+    //todo isbr: fix
```

Comment:
Any chance are the failing tests failing due to the resource not existing? If this was an issue I'd expect all the tests using audience to fail but they don't.

Reply:
Both defaultAudience and audienceError pass. storageAccountAudience fails with this error: 
```
FileSystemApiTests.storageAccountAudience:2239 » DataLakeStorage Status code 401, "?<?xml version="1.0" encoding="utf-8"?><Error><Code>InvalidAuthenticationInfo</Code><Message>Server failed to authenticate the request. Please refer to the information in the www-authenticate header.
RequestId:906cf234-701e-007b-7118-01d5fa000000
Time:2023-10-17T16:36:38.6607183Z</Message><AuthenticationErrorDetail>Audience validation failed. Audience did not match.</AuthenticationErrorDetail></Error>"
```